src/devSVG.cpp

```diff
@@ -241,55 +290,59 @@ void svg_clip(double x0, double x1, double y0, double y1, pDevDesc dd) {
 }
 
 void svg_new_page(const pGEcontext gc, pDevDesc dd) {
+BEGIN_RCPP
```

Comment:
I see. I did that since I saw there was a discussion by @kevinushey: http://lists.r-forge.r-project.org/pipermail/rcpp-devel/2015-March/008501.html

> Calling `Rf_error()` directly from C++
can result in undefined behaviour -- this is because calling
`Rf_error()` will result in a `longjmp`, which bypasses the
destructors of any objects on the stack (thereby potentially leaking
memory, but in general, causing undefined behaviour).